specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2022-09-08-preview/managedccf.json

```diff
@@ -0,0 +1,555 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "type": "string",
+            "description": "The filter to apply on the list operation. eg. $filter=ledgerType eq 'Public'",
+            "name": "$filter",
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; response contains the list of Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCFList"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure. 404 - NotFound (Managed CCF does not exist.)",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ManagedCCFListBySub": {
+            "$ref": "./examples/Mccf_ListBySub.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceLocation": {
+      "description": "Location of the ARM Resource",
+      "type": "object",
+      "properties": {
+        "location": {
+          "description": "The Azure location where the Confidential Ledger is running.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      }
+    },
+    "Tags": {
+      "description": "Tags for Confidential Ledger Resource",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Additional tags for Confidential Ledger",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ]
+        }
+      }
+    },
+    "Resource": {
+      "description": "An Azure resource.",
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "Fully qualified resource Id for the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "systemData": {
+          "description": "Metadata pertaining to creation and last modification of the resource",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceProviderOperationDefinition": {
```

Comment:
>ResourceProviderOperationDefinition [](http://example.com/codeflow?start=5&length=35)

This isn't used, please remove. If you need it in the future, use the Operation definition from v3 common types.